app/admin/global_country_platform/adaptation.rb

```diff
@@ -0,0 +1,87 @@
+ActiveAdmin.register_page 'Global Cw Platform Adaptation' do
+  section_name = 'adaptation'
+  platform_name = 'global_cw_platform'
+  s3_folder_path = "#{CW_FILES_PREFIX_TEST}adaptation"
+
+  menu parent: 'Climate Watch Global', label: section_name.capitalize
+
+  content do
+    render partial: 'admin/form_upload_datasets', locals: {
+      datasets: Admin::Dataset.joins(:section).
+        where(sections: {name: section_name}).
+        where(sections: {platform_id: Admin::Platform.find_by(name: platform_name).id}),
+      upload_path: admin_global_cw_platform_adaptation_upload_datafile_path,
+      download_path: admin_global_cw_platform_adaptation_download_datafiles_path,
+      download_single_data_file_path: admin_global_cw_platform_adaptation_download_datafile_path,
+      import_path: admin_global_cw_platform_adaptation_run_importer_path
+    }
+  end
+
+  page_action :upload_datafile, method: :post do
+    dataset = Admin::Dataset.find(params[:dataset_id])
+
+    if params[:datafile].content_type != 'text/csv'
```

Comment:
this breaks when you hit "upload" without selecting a file, not a major issue but best avoid a 500

Reply:
thanks for catching that! I'll fix it now

Reply:
fixed!